who-metrics-ui/src/components/RepositoriesTable.tsx

```diff
@@ -565,6 +612,14 @@ const RepositoriesTable = () => {
     return sortedRows;
   };
 
+  const testTimeBasedFilter = (minDays: number | undefined, maxDays: number | undefined, timeInMs: number) => {
```

Comment:
I didn't test it, but my hope is this will be reusable for `openIssueAverageAge`, `issueTimeToResponse`, etc.

Reply:
YES